apiserver/facades/controller/sshserver/register.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package sshserver
+
+import (
+	"reflect"
+
+	"github.com/juju/errors"
+
+	apiservererrors "github.com/juju/juju/apiserver/errors"
+	"github.com/juju/juju/apiserver/facade"
+)
+
+// Register is called to expose a package of facades onto a given registry.
+func Register(registry facade.FacadeRegistry) {
+	registry.MustRegister("SSHServer", 1, func(ctx facade.Context) (facade.Facade, error) {
+		return NewExternalFacade(ctx)
+	}, reflect.TypeOf((*Facade)(nil)))
+}
+
+// NewExternalFacade creates a new authorized Facade.
+func NewExternalFacade(ctx facade.Context) (*Facade, error) {
```

Comment:
I'd just call this NewFacade

Reply:
I gottan this idea from another facade aha because I need a "NewFacade" for tests, then realised it's not so easy to test...

Reply:
You can't because it requires mocks that reside in the test package, i'd need to flip the package of the mocks...